classes/class-ep-debug-bar-elasticpress.php

```diff
@@ -162,7 +162,7 @@ public function render() {
 						<?php endif; ?>
 						<a class="copy-curl" data-request="<?php echo esc_attr( addcslashes( $curl_request, '"' ) ); ?>">Copy cURL Request</a>
```

Comment:
ooo non translatable strings! Bet you can tell what I'm about to ask...

Reply:
@roborourke what textdomain should I use?

Reply:
Yeah `debug-bar` is good